docs_source_files/content/support_packages/working_with_colours.en.md

```diff
@@ -151,22 +151,22 @@ Color loginButtonColour = Color.fromString(driver.findElement(By.id("login")).ge
 Color loginButtonBackgroundColour = Color.fromString(driver.findElement(By.id("login")).getCssValue("background-color"));
   {{< / code-panel >}}
   {{< code-panel language="python" >}}
-login_button_colour = driver.find_element_by_id('login').value_of_css_property('color')
-login_button_background_colour = driver.find_element_by_id('login').value_of_css_property('background-color');
+login_button_colour = Color.from_string(driver.find_element_by_id('login').value_of_css_property('color'))
+login_button_background_colour = Color.from_string(driver.find_element_by_id('login').value_of_css_property('background-color'))
```

Comment:
Hi @praveen4463,

I think there is a bit of misunderstanding. As @diemol suggested, he is referring to other translated pages like fr.md, es.md, zh-ch.md, de.md,jo.md, ka.md, nl.md etc and not for other code samples.

For example, code changes for java should be changed in a few of the below documents too. 
https://www.selenium.dev/documentation/zh-cn/support_packages/working_with_colours/
https://www.selenium.dev/documentation/de/support_packages/working_with_colours/
https://www.selenium.dev/documentation/fr/support_packages/working_with_colours/
https://www.selenium.dev/documentation/es/support_packages/working_with_colours/

Please let me know if you need any information, we can help you out.

Thanks,
Harsha